cmd/cmd.go

```diff
@@ -64,8 +64,10 @@ func pull(model string) error {
 }
 
 func RunGenerate(_ *cobra.Command, args []string) error {
+	// join all args into a single prompt
+	prompt := strings.Join(args[1:], " ")
```

Comment:
Better to keep this simple and make it one arg vs joining - the user can it in quotes:

```
ollama run nous-hermes "$(cat input.txt) please summarize this story"
```

This will allow us to use the addl' args for more context (e.g. shell expansion w/ files)

```
ollama run starcoder "where can I simplify this code?" *.go
```

For example, check out how `echo` works:

```
% echo llama/*.go hi
llama/llama.go llama/llama_cublas.go llama/llama_metal.go llama/llama_openblas.go llama/options.go hi
```

Reply:
On second thought, as-is is pretty good, users can just put arbitrary arguments after `ollama run <model>` and it will be passed in as context, and that's a decent experience